src/Reshuffle.ts

```diff
@@ -60,13 +60,12 @@ export default class Reshuffle {
   }
 
   registerHTTPDelegate(path: string, delegate: BaseHttpConnector): Reshuffle {
-    let httpMultiplexer = this.httpDelegates[path]
+    const httpMultiplexer = this.httpDelegates[path] || new HttpMultiplexer(path)
```

Comment:
If this is const then the assignment in a few lines fails.

Reply:
line 65 and 67 needs to be deleted, catered by the `|| new HttpMultiplexer(path)`, and so the const will remain ;)

Reply:
@lagache  see build error:
src/Reshuffle.ts:66:7 - error TS2588: Cannot assign to 'httpMultiplexer' because it is a constant.

Reply:
@ashevat yes I know, sorry I made one mistake in my comment above, line 65 **TO** 67 need to be removed.
the if is replaced by the `|| new HttpMultiplexer(path)`
@duckranger do you confirm?